pkgdb/src/buildenv/assets/etc/profile.d/0800_cuda.sh

```diff
@@ -0,0 +1,38 @@
+# shellcheck shell=bash
+export _coreutils="@coreutils@"
+export _gnused="@gnused@"
+export _findutils="@findutils@"
+# ============================================================================ #
+#
+# Setup CUDA
+#
+# ---------------------------------------------------------------------------- #
+
+# Only run if FLOX_ENV_ENABLE_CUDA feature flag is set
+activate_cuda(){
+  if [[ "$FLOX_FEATURES_ENV_ENABLE_CUDA" != 1 ]]; then
+    return 0
+  fi
+
+  if ! ( "$_findutils/bin/find" /dev -maxdepth -iname 'nvidia*' | read -r ;); then
```

Comment:
blocking question: does maxdepth require an argument? Not sure if I have a different version but I'm seeing
```
find: Expected a positive decimal integer argument to -maxdepth, but got ‘-iname’
```

Reply:
Agreed.... i'm not sure how that got dropped, fixed.